app/models/comment.rb

```diff
@@ -24,6 +24,13 @@ class Comment < ApplicationRecord
   include Rails.application.routes.url_helpers
   include LinkToHelper
 
+  CREATION_RATE_INTERVALS = {
+    1 => 2.seconds,
+    2 => 5.minutes,
+    4 => 30.minutes,
+    6 => 1.hour
+  }
```

Comment:
Not clear what the key/values represent here. How about making this an array of hashes with keys `limit` & `duration`

Reply:
Renamed to `DEFAULT_CREATION_RATE_LIMITS` (and set via a `cattr_accessor`).

I think the improved constant name should help – I read it as "Limit of 2 records in 5 minutes". I think this makes a clearer definition than adding extra keys, but happy to reconsider in https://github.com/mysociety/alaveteli/issues/7620.